sailon/random_novelty_detector.py

```diff
@@ -10,10 +8,55 @@
 
 from smqtk_core import Configurable, Pluggable
 
+from .errors import RoundError
+from .novelty_interface import ONDAdapter
```

Comment:
Could you rename the module to ond_adaptor?

Reply:
Updated the module name.